packages/ui-menu/src/Menu/MenuItemGroup/index.js

```diff
@@ -245,11 +258,11 @@ class MenuItemGroup extends Component {
   render() {
     const props = omitProps(this.props, MenuItemGroup.propTypes)
     return (
-      <span {...props} className={styles.root} role="presentation">
+      <span {...props} css={this.props.styles.root} role="presentation">
```

Comment:
this should be `this.props.styles.menuItemGroup`?

Reply:
Lol, nice catch